modules/cleaning-crio-storage.adoc

```diff
@@ -72,6 +72,8 @@ $ sudo -i
 [source, terminal]
 +
 ----
+# Non-HostNetwork pods must be removed first, as their removal relies on the networking plugin pods, which are HostNetwork.
+# for pod in $(crictl pods -q); do if [[ "$(crictl inspectp $pod | jq -r .status.linux.namespaces.options.network)" != "NODE" ]]; then crictl rmp -f $pod; fi; done
 # crictl rmp -fa
 ----
```

Comment:
```suggestion
. Stop the containers and pods:

.. Stop the non-HostNetwork pods first, because their removal relies on the networking plugin pods, which are HostNetwork.
+
[source,terminal]
----
# for pod in $(crictl pods -q); do if [[ "$(crictl inspectp $pod | jq -r .status.linux.namespaces.options.network)" != "NODE" ]]; then crictl rmp -f $pod; fi; done
----

.. Stop all of the other pods:
+
[source,terminal]
----
# crictl rmp -fa
----
```

Reply:
fixed!